ReactSkia/sdk/CurlNetworking.cpp

```diff
@@ -176,20 +173,26 @@ size_t CurlNetworking::progressCallbackCurlWrapper(void *clientp, double dltotal
 } 
 size_t CurlNetworking::headerCallbackCurlWrapper(char* buffer, size_t size, size_t nitems, void* userData) {
   CurlRequest *curlRequest = (CurlRequest *)userData;
-  curlRequest->curlResponse.headerBuffer  = (char *) realloc(curlRequest->curlResponse.headerBuffer , curlRequest->curlResponse.headerBufferOffset+nitems+1); 
-  RNS_LOG_ASSERT(curlRequest->curlResponse.headerBuffer  , "headerBuffer cannot be null");
-  // Each headerInfo line comes as a seperate callback, so append all here
-  memcpy(&(curlRequest->curlResponse.headerBuffer[curlRequest->curlResponse.headerBufferOffset]), buffer, nitems);
-  curlRequest->curlResponse.headerBufferOffset+= nitems;
+  // Each headerInfo line comes as a seperate callback
```

Comment:
remove this changes

Reply:
I would suggest this file to be submitted as separate CL for Network moduel